### ISSUE TITLE
Issue with backing up "/" and bup's "--strip-path /" option

### ISSUE THREAD

**Issue by blueyed** [anarcat/bup-cron#16]
I've just noticed that backing up "/" appeared to strip off all paths of the first level.

This has been fixed in https://github.com/bup/bup/commit/1e6960cc99f293c740d6ab4c9f0bd4008f5626d2 - I don't know if it has regressed recently or has been like this for longer.

Maybe bup-cron could not use `--strip-path` when backing up `/`?

**Comment by anarcat** [comment]
oh boy, that old bug. i had tons of problems with `--strip-path`, but i thought they were all fixed. maybe it's because i just didn't notice the problem again.

i'm hesitant in adding workarounds for bup bugs in bup-cron. shouldn't we make people upgrade instead?

**Comment by blueyed** [comment]
> i'm hesitant in adding workarounds for bup bugs in bup-cron. shouldn't we make people upgrade instead?

Agreed.  But maybe a warning would be good, in case it's easy to detect this scenario.

**Comment by blueyed** [comment]
Maybe `--strip-path` could just be skipped if it's `/`?
